antsibull/data/docsite/role.rst.j2

```diff
@@ -0,0 +1,286 @@
+.. Document meta
+
+:orphan:
+
+{# If we can put together source and github repo, we could make the Edit me of github button work.
+   See meta.get("source") in Ansible's docs/docsite/_themes/sphinx_rtd_theme/breadcrumbs.html
+   for more information
+:source: @{ source }@
+-#}
+
+.. Anchors
+
+.. _ansible_collections.@{plugin_name}@_@{plugin_type}@:
```

Comment:
Is this a list of plugins contained in the same collection?

Reply:
Here `@{plugin_name}@` is the FQCN of the role, and `@{plugin_type}@ == 'role'`. This is just the main label for this role, there's no list here.

Reply:
ah, I see - thanks